test/server/cards/01_SOR/units/WedgeAntillesStarOfTheRebellion.spec.ts

```diff
@@ -0,0 +1,55 @@
+describe('Wedge Antilles, Star of the Rebellion', function() {
+    integration(function(contextRef) {
+        describe('Wedge Antilles\'s ability', function() {
+            beforeEach(function () {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        hand: ['alliance-xwing', 'battlefield-marine', 'clan-saxon-gauntlet'],
+                        groundArena: ['wedge-antilles#star-of-the-rebellion'],
+                        leader: 'hera-syndulla#spectre-two',
```

Comment:
I think there's some stuff in here that isn't relevant to the test such as the Hera leader and the explicit resource count (it sets at 20 by default). We try to keep the test setup to the bare minimum just for readability, do you mind removing the unused stuff?